spec/class_album_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../class_musicalbum_handler'
+require_relative '../class_genres_handler'
+
+RSpec.describe MusicalbumHandler do
+  let(:musicalbum_handler) { MusicalbumHandler.new }
```

Comment:
- [ ] Please make sure to add a test for the method `can_be_archived?()` in the MusicAlbum class. :pray:

Reply:
thank you. @MoyasiGinko is goint to fixit

Reply:
Thank you. class_album_can_be_archived_spec.rb added! 👍🏽